src/index.ts

```diff
@@ -196,7 +195,7 @@ const replaceVariables = (obj, variables) => {
         ) {
             const variableName = value.replace(isVariableDropinConst, "")
             return variables[variableName]
-        } else if (isObject(value) && !isArray(value)) {
+        } else if (isObject(value) && !Array.isArray(value)) {
```

Comment:
I used here `Array.isArray` again. Having a helper function which simply maps to another standard library function seemed a bit round-about.

Reply:
Agree